extensions/wikia/RTE/RTE.class.php

```diff
@@ -427,23 +431,14 @@ private static function checkEditorConditions() {
 	/**
 	 * Disable CK editor - MediaWiki editor will be loaded
 	 */
-	public static function disableEditor($reason = false) {
+	public static function disableEditor(string $reason) {
```

Comment:
Let's use spaces within parentheses per [the MediaWiki PHP coding conventions](https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP).